SPEC.md

```diff
@@ -104,44 +106,42 @@ The test index makes use of symbolic language in describing connection and messa
     <- pong response with the same `sequence` number
 
 ### ZG-CONFORMANCE-004
```

Comment:
I think this case can be written more succinctly, see e.g. https://github.com/runziggurat/zcash/blob/main/SPEC.md#zg-conformance-007

Reply:
Done